modules/nw-using-nodeport.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * ingress/getting-traffic-cluster.adoc
+
+[id="nw-using-nodeport_{context}"]
+= Using a NodePort to get traffic into the cluster
+
+Use a `NodePort`-type `Service` resource to expose a service on a specific port
```

Comment:
Should this instance of _NodePort_ have formatting? The instances that follow do not.